enf-reducer/cut-unnecessary-and.metta

```diff
@@ -0,0 +1,28 @@
+;(: AndCut (-> Expression Expression))
```

Comment:
Can you rename the function name to `andCut` to conform to our convention? we use `camelCase` for function and variable names and use `PascalsCase` for type names.

Reply:
Also add a few lines of comments about what the `andCut` function is :).